panaroo/integrate.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
+"""
+A script to merge a single GFF file with a pre-existing panaroo output
+"""
+
+import os
+import tempfile
+import networkx as nx
+import subprocess
+import shutil
+import numpy as np
+
+from .__init__ import __version__
+from .prokka import process_prokka_input
+from .cdhit import run_cdhit
+from .generate_network import generate_network
+from .isvalid import *
+
+def get_options(): #options for integrating (combination of merge graph and cdhit options)
+   
+    import argparse
+
+    description = 'Integrate new gff file into pre-existing graph'
+    parser = argparse.ArgumentParser(description=description,
+                                     prog='panaroo_integrate')
```

Comment:
You should also add this script entry point to `setup.py` / the conda `meta.yml` so that it is installed with the rest of the package